byron/ledger/formal-spec/blockchain-interface.tex

```diff
@@ -49,6 +49,57 @@ \subsubsection{Delegation interface rules}
 \label{sec:delegation-interface-rules}
 
 \begin{figure}[htb]
+  \begin{equation}
+    \label{eq:rule:delegation-interface}
+    \inference
+    {
+      {\left(\begin{array}{l}
+       \mathcal{K}\\
+        e\\
+        s
+      \end{array}\right)}
+      \vdash
+      \trans{sdeleg}{}
+      {\left(
+        \begin{array}{l}
+          \epsilon\\
```

Comment:
Here I think it is better to make these variables, e.g. `sds_0` and `eks_0, so that we abstract away from what the initial rules actually generate (in this way, if we ever change the initial rules we don't need to change this one ;) ). Does that make sense?

Reply:
:+1:

Reply:
fixed - see the updated init rules in Fig 14/36